README.md

```diff
@@ -279,6 +280,20 @@ And then run `bundle update redis`.
 
 Note that the Hyrax Management Guide contains additional information on [how to configure ActionCable in production environments](https://github.com/samvera/hyrax/wiki/Hyrax-Management-Guide#notifications).
 
+## Set id validation
+
+Hyrax uses [noid-rails](https://github.com/samvera/noid-rails) for id generation.  By default, the minter will not validate whether a newly minted id is already in use.
+
+To set up id validation, add `config/initializers/noid_rails.rb`
+
+```
+::Noid::Rails.config.identifier_in_use = lambda { |id| ActiveFedora::Base.exists?(id) || ActiveFedora::Base.gone?(id) }
+```
+
+Requirement: noid-rails >= 3.0.1 (older versions prevented the override of identifier_in_use)
```

Comment:
Shouldn't this be in the gemspec?

Reply:
:+1: I don't see any reason to not bump it in the gemspec since it is already at `~> 3.0.0`.